multisig/proposals/sepolia/vip-converter/vipConverter.ts

```diff
@@ -0,0 +1,96 @@
+import { acceptOwnershipCommandsAllConverters, callPermissionCommandsAllConverter } from "./commands";
+import {
+  converterCommands,
+  grant,
+  incentiveAndAccessibilityForUSDCPrimeConverter,
+  incentiveAndAccessibilityForUSDTPrimeConverter,
+  incentiveAndAccessibilityForWBTCPrimeConverter,
+  incentiveAndAccessibilityForWETHPrimeConverter,
+  incentiveAndAccessibilityForXVSVaultConverter,
+} from "./commands";
+
+export const XVS_VAULT_TREASURY = "0xCCB08e5107b406E67Ad8356023dd489CEbc79B40";
+export const PROTOCOL_SHARE_RESERVE_PROXY = "0xbea70755cc3555708ca11219adB0db4C80F6721B";
+export const VTREASURY = "0x4116CA92960dF77756aAAc3aFd91361dB657fbF8";
+
+const vipConverter = () => {
+  return makeProposal([
+    ...acceptOwnershipCommandsAllConverters,
+    {
+      target: XVS_VAULT_TREASURY,
+      signature: "acceptOwnership()",
+      params: [],
+    },
+    {
+      target: CONVERTER_NETWORK,
+      signature: "acceptOwnership()",
+      params: [],
+    },
+    ...callPermissionCommandsAllConverter,
+
+    grant(CONVERTER_NETWORK, "addTokenConverter(address)", GUARDIAN),
+    grant(CONVERTER_NETWORK, "removeTokenConverter(address)", GUARDIAN),
+    grant(XVS_VAULT_TREASURY, "fundXVSVault(uint256)", GUARDIAN),
+
+    {
+      target: PROTOCOL_SHARE_RESERVE_PROXY,
+      signature: "addOrUpdateDistributionConfigs((uint8,uint16,address)[])",
+      params: [
+        [
+          [0, 6000, VTREASURY],
+          [0, 2000, XVS_VAULT_CONVERTER],
+          [0, 200, USDC_PRIME_CONVERTER],
+          [0, 200, USDT_PRIME_CONVERTER],
+          [0, 100, WBTC_PRIME_CONVERTER],
+          [0, 1500, WETH_PRIME_CONVERTER],
+          [1, 8000, VTREASURY],
+          [1, 2000, XVS_VAULT_CONVERTER],
+        ],
+      ],
+    },
+    ...converterCommands,
+    {
+      target: USDT_PRIME_CONVERTER,
+      signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+      params: [BaseAssets[0], USDTPrimeConverterTokenOuts, incentiveAndAccessibilityForUSDTPrimeConverter],
+    },
+    {
+      target: USDC_PRIME_CONVERTER,
+      signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+      params: [BaseAssets[1], USDCPrimeConverterTokenOuts, incentiveAndAccessibilityForUSDCPrimeConverter],
+    },
+    {
+      target: WBTC_PRIME_CONVERTER,
+      signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+      params: [BaseAssets[2], WBTCPrimeConverterTokenOuts, incentiveAndAccessibilityForWBTCPrimeConverter],
+    },
+    {
+      target: WETH_PRIME_CONVERTER,
+      signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+      params: [BaseAssets[3], WETHPrimeConverterTokenOuts, incentiveAndAccessibilityForWETHPrimeConverter],
+    },
+    {
+      target: XVS_VAULT_CONVERTER,
+      signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+      params: [BaseAssets[4], XVSVaultConverterTokenOuts, incentiveAndAccessibilityForXVSVaultConverter],
+    },
+  ]);
```

Comment:
I miss the commands to enable the Prime markets. Based on the commands included in the VIP-201, I think these are the missing commands (please, check it):

```
  {
    target: COMPTROLLER_LST,
    signature: "setPrimeToken(address)",
    params: [PRIME],
  },
  {
    target: PRIME_LIQUIDITY_PROVIDER,
    signature: "initializeTokens(address[])",
    params: [[ETH, BTC, USDC, USDT]],
  },
  {
    target: PRIME_LIQUIDITY_PROVIDER,
    signature: "setTokensDistributionSpeed(address[],uint256[])",
    params: [
      [ETH, BTC, USDC, USDT],
      [0, 0, 0, 0],
    ],
  },
  {
    target: PRIME,
    signature: "addMarket(address,uint256,uint256)",
    params: [vETH, ethers.utils.parseEther("2"), ethers.utils.parseEther("4")],
  },
  {
    target: PRIME,
    signature: "addMarket(address,uint256,uint256)",
    params: [vBTC, ethers.utils.parseEther("2"), ethers.utils.parseEther("4")],
  },
  {
    target: PRIME,
    signature: "addMarket(address,uint256,uint256)",
    params: [vUSDC, ethers.utils.parseEther("2"), ethers.utils.parseEther("4")],
  },
  {
    target: PRIME,
    signature: "addMarket(address,uint256,uint256)",
    params: [vUSDT, ethers.utils.parseEther("2"), ethers.utils.parseEther("4")],
  },
  {
    target: PRIME,
    signature: "setLimit(uint256,uint256)",
    params: [
      0, // irrevocable
      500, // revocable
    ],
  },
  {
    target: PRIME_LIQUIDITY_PROVIDER,
    signature: "pauseFundsTransfer()",
    params: [],
  },
```

Reply:
@chechu, the function to add market accepts comptroller as an argument too. So which comptroller and markets, we should consider? Should we add all comptrollers with their markets containing markets from the following ones:
USDC, USDT, WETH, WBTC?

Reply:
we also need to set prime address in **COMPTROLLER_CURVE** and **COMPTROLLER_STABLE_COINS**.

Reply:
Done https://github.com/VenusProtocol/vips/pull/303/commits/7fde183c32cb94c8e7ec50f3c01262679d15c4fe

Reply:
PLP is already paused. So, I have removed that command.